template-for-repository/proofs/Makefile.common

```diff
@@ -938,6 +938,10 @@ echo-proof-uid:
 echo-project-name:
 	@echo $(PROJECT_NAME)
 
+.PHONY: echo-litani-path
+echo-litani-path:
```

Comment:
It actually looks like there's already a rule called `litani-path` that does this? No need for this change. 🧲